tests/acceptance/features/bootstrap/FeatureContext.php

```diff
@@ -794,6 +794,13 @@ public function getBaseUrl(): string {
 		return $this->baseUrl;
 	}
 
+	/**
+	 * @return string
+	 */
+	public function getStorageUserOcisRoot() : string {
+		return getenv('STORAGE_USERS_OCIS_ROOT') ? getenv('STORAGE_USERS_OCIS_ROOT') : getenv('HOME') . "/.ocis/storage/users";
+	}
```

Comment:
we might want to have these envs checks
```suggestion
		return $STORAGE_USERS_OCIS_ROOT if set
		return $OCIS_BASE_DATA_PATH/storage/users if set
		or return $HOME/.ocis/storage/users

	}
```